traffic-sniffer/lib/sniffer-manager/mock-manager-controller.ts

```diff
@@ -127,6 +254,30 @@ export class MockManagerController {
       }
     );
 
+    /**
+     * @openapi
+     * /sharkio/sniffer/:port/mock/actions/activate:
+     *   post:
+     *     tags:
+     *      - mock
+     *     description: Activate a mock
+     *     parameters:
+     *       - name: port
+     *         in: query
+     *         description: service port
+     *         required: true
+     *     requestBody:
+     *        description: Activate a mock
+     *        content:
+     *          application/json:
+     *            schema:
+     *              type: object
```

Comment:
Is the request body mandatory in this doc? I mean - all the function body references is the params object